pypreprocess/subject_data.py

```diff
@@ -39,14 +40,23 @@
 segment_acronyms = ("Acronyms: TPM means Tissue Probability Map; GM means "
                    "Grey-Matter;"
                    " WM means White-Matter; CSF means Cerebro-Spinal Fuild")
-cv_tc_tooltip = ("Coefficient of Variation (CoV) of the BOLD signal. "
-                 "The Coefficient of "
-                 "Variation is defined as the variance of the BOLD signal "
-                 "(over the voxels in the brain volume) divided by the mean "
-                 "thereof. Generally, if the CoV curve is below 1% (.01), then"
-                 " it's OK. Also, the CoV typically spikes at the end and the "
-                 "begining of the acquisition, due to saturation effects, "
-                 "scanner instability, etc.")
+tsdiffana_tooltips = [
+    ("(Squared) differences across sequential volumes. "
+     "A large value indicates an artifact "
+     "that occurred during the slice acquisition, possibly related to motion."),
+    ("Average signal over each volume. A large drop/peak (e.g. 5%) indicates "
+     "an artefact."),
+    ("Variance index per slice. Note that aqquisition artifacts can be slice"
+     "-specific. Look at the day if there is a peak somewhere."),
```

Comment:
@bthirion "Look at the day" :confused:. Should it be "Look at the data instead"?